Documentation/ceph-quickstart.md

```diff
@@ -96,6 +96,55 @@ rook-ceph-osd-1-5b96b56684-r7zsp       1/1       Running     0          1m
 rook-ceph-osd-prepare-mynode-ftt57     0/1       Completed   0          1m
 ```
 
+# Support for authenticated docker registries
+
+If you want to use an image from authenticated docker registry (e.g. for image cache/mirror), you'll need to 
+add an `imagePullSecret` to all relevant service accounts. This way all pods created by the operator (for service account: 
+rook-ceph-system) or all new pods in the namespace (for service account: default) will have the `imagePullSecret` added
+to their spec.
+
+So first, we'll create the secret as described [here](https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod):
+
+```bash
+# for namespace rook-ceph-system (operator)
+kubectl -n rook-ceph-system create secret docker-registry my-registry-secret --docker-server=DOCKER_REGISTRY_SERVER --docker-username=DOCKER_USER --docker-password=DOCKER_PASSWORD --docker-email=DOCKER_EMAIL
+
+# and for namespace rook-ceph (cluster)
+kubectl -n rook-ceph create secret docker-registry my-registry-secret --docker-server=DOCKER_REGISTRY_SERVER --docker-username=DOCKER_USER --docker-password=DOCKER_PASSWORD --docker-email=DOCKER_EMAIL
+```
+
+Next we'll add the following snippet to all relevant service accounts as described [here](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#add-imagepullsecrets-to-a-service-account):
+
+```yaml
+imagePullSecrets:
+- name: my-registry-secret
+```
+
+The service accounts are:
```

Comment:
there is also the `rook-ceph-cluster` serviceaccount in the `rook-ceph` namespace, currently only used by OSDs